tools/devops/device-tests/cambridge-ios-32b-device-tests.yml

```diff
@@ -0,0 +1,31 @@
+# Runs the device tests on iOS 32b devices.
+parameters:
+- name: disableProviCleanup
+  type: boolean
+  default: false
+
+- name: clearProviCache
+  type: boolean
+  default: false
```

Comment:
This part is repeated in every yml scripts (and always identical), can it be moved into a template instead?

Reply:
parameters do not support templates as variables do :/

Reply:
For reference, the schema definition is:
```yml
parameters:
- name: string          # name of the parameter; required
  type: enum            # data types, see below
  default: any          # default value; if no default, then the parameter MUST be given by the user at runtime
  values: [ string ]    # allowed list of values (for some data types)
  secret: bool          # whether to treat this value as a secret; defaults to false
```